theme/index.hbs

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE HTML>
-<html lang="{{ language }}" class="sidebar-visible no-js {{ default_theme }}">
+<html lang="{{ language }}" dir="rtl" class="sidebar-visible no-js {{ default_theme }}">
```

Comment:
We cannot include this unconditionally — it will also affect the English pages then.

We will need some custom logic in the `public.yml` file: I'm thinking we can add `dir="rtl"` for the RTL languages at this point. We will do this in a separate PR — please revert it here so that this PR is only focused on the Persian translation.

Reply:
```suggestion
<html lang="{{ language }}" class="sidebar-visible no-js {{ default_theme }}">
```

Reply:
OK. I will continue the Persian translation until the problem of right-to-left interference with other languages is solved.

Reply:
Sounds great, thanks! It will take a while to translate the course, so we'll figure out how to publish it nicely in that time :smile: